KarhooUISDK/CommonUI/VehicleCapacityView/NewVehicleCapacityView.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Created by Bartlomiej Sopala on 10/03/2022.
+// Copyright (c) 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import KarhooSDK
+
+public struct KHNewVehicleCapacityViewID {
+    public static let capacityView = "vehicle_capacity_view"
+    public static let passengerStack = "passenger_stack"
+    public static let baggageStack = "baggage_stack"
+    public static let baggageIcon = "baggage_image"
+    public static let baggageCapacityLabel = "baggage_capacity_label"
+    public static let capacityIcon = "passenger_capacity_image"
+    public static let passengerCapacityLabel = "passenger_capacity_label"
+}
+
+final class NewVehicleCapacityView: UIStackView {
+
+    private lazy var passengerStack = UIStackView().then {stack in
+        stack.backgroundColor = KarhooUI.colors.background1
+        stack.accessibilityIdentifier = KHNewVehicleCapacityViewID.passengerStack
+        stack.clipsToBounds = true
+        stack.layer.cornerRadius = UIConstants.CornerRadius.xSmall
+        stack.isLayoutMarginsRelativeArrangement = true
+        stack.directionalLayoutMargins = NSDirectionalEdgeInsets(
+            top: UIConstants.Spacing.xxSmall,
+            leading: UIConstants.Spacing.xSmall,
+            bottom: UIConstants.Spacing.xxSmall,
+            trailing: UIConstants.Spacing.xSmall
+        )
+        stack.spacing = UIConstants.Spacing.xxSmall
+    }
+
+    private lazy var baggageStack = UIStackView().then {stack in
+        stack.accessibilityIdentifier = KHNewVehicleCapacityViewID.baggageStack
+        stack.backgroundColor = KarhooUI.colors.background1
+        stack.clipsToBounds = true
+        stack.layer.cornerRadius = UIConstants.CornerRadius.xSmall
+        stack.isLayoutMarginsRelativeArrangement = true
+        stack.directionalLayoutMargins = NSDirectionalEdgeInsets(
+            top: UIConstants.Spacing.xxSmall,
+            leading: UIConstants.Spacing.xSmall,
+            bottom: UIConstants.Spacing.xxSmall,
+            trailing: UIConstants.Spacing.xSmall
+        )
+        stack.spacing = UIConstants.Spacing.xxSmall
+    }
+
+    private var baggageImageView: UIImageView = {
```

Comment:
Why do you use `.then` syntax in two views, closure initializer in another one and lazy initializer in the rest of them. Could you please try to make it somehow more consistent?